taxonomies/categories.txt

```diff
@@ -53144,6 +53219,40 @@ en:Cassoulets from Toulouse
 fr:Cassoulets toulousains
 nl:Cassoulet uit Toulouse
 
+<en:Duck dishes
+<en:Cassoulets
+en:Cassoulets with duck confit
+fr:Cassoulets au confit de canard
+nl:Cassoulet met geconfijte eend
+agribalyse_proxy_food_code:en:25099
+agribalyse_proxy_food_name:en:White bean stew, with duck or goose, canned
```

Comment:
why agribalyse_proxy instead of just agribalyse?

Reply:
if it's to make a distinction between cassoulet with duck and cassoulet with duck confit, then we should just merge them. The duck is always duck confit in cassoulet.

Reply:
merged